docs/ops/movement/StridedSlice_1.md

```diff
@@ -51,19 +51,19 @@
 
 **Inputs**:
 
-*   **1**: Multidimensional input tensor to be sliced. Required.
+*   **1**: input tensor to be sliced of type `T` and arbitrary shape. **Required.**
 
-*   **2**: `begin` input - 1D input tensor with begin indexes for input tensor slicing. Required.
-           Out-of-bounds values are silently clamped. If `begin_mask[i]` is 1, the value of `begin[i]` is ignored
-           and the range of the appropriate dimension starts from 0.
-           Negative values mean indexing starts from the end. For example, if `foo=[1,2,3]`, `begin[0]=-1` means `begin[0]=3`.
+*   **2**: `begin` - 1D tensor of type `T_IND` with begin indexes for input tensor slicing. **Required.**  
+    Out-of-bounds values are silently clamped. If `begin_mask[i]` is `1`, the value of `begin[i]` is ignored and the range of the appropriate dimension starts from `0`. Negative values mean indexing starts from the end. For example, if `foo=[1,2,3]`, `begin[0]=-1` means `begin[0]=3`.
```

Comment:
Let's add a name fo the tensor with data and use this name instead of `foo` in the examples. Otherwise, examples are non-obvious.

Reply:
Added in this [commit](https://github.com/openvinotoolkit/openvino/pull/5887/commits/a26aa5c449d375d3f8ba8dc1d0576cc2051f6bce).